### PR TITLE
Feat: Auto close logic for Inactive User

### LINKED ISSUE #16: Add auto close logic
If the user is inactive for more than 60s
Probably needs to be configurable

### PR DESCRIPTION
**Pull request by Airbornharsh**
issue close #16 

1. Using timers in Game class to store the logic to destroy itself and send signals to user
2. Removing the game from the games
3. Reflecting the Game Over part in the UI

### CODE REVIEW

**Comment by Airbornharsh** [comment]
There is issue regarding formatting , so can check #19 
I have added prettier which can help to persist a style in the code

File: backend1/src/Game.ts [hunk 1]
```diff
@@ -77,4 +78,33 @@ export class Game {
         }
         this.moveCount++;
     }
+
+    gameOver(socket: WebSocket) {
+        console.log("game over")
```
Comment (hkirat) [inline backend1/src/Game.ts:83]: remove

File: backend1/src/Game.ts [hunk 2]
```diff
@@ -77,4 +78,33 @@ export class Game {
...
+        this.player1.send(JSON.stringify({
+            type: GAME_OVER,
+            payload: {
+                winner: this.board.turn() === "w" ? "black" : "white"
+            }
+        }))
+        // this.player1.close();
```
Comment (hkirat) [inline backend1/src/Game.ts:90]: remove comment?

File: backend1/src/Game.ts [hunk 3]
```diff
@@ -77,4 +78,33 @@ export class Game {
...
+        this.player2.send(JSON.stringify({
+            type: GAME_OVER,
+            payload: {
+                winner: this.board.turn() === "w" ? "black" : "white"
+            }
+        }))
+        // this.player2.close();
```
Comment (hkirat) [inline backend1/src/Game.ts:97]: remove

**Review by hkirat — CHANGES_REQUESTED** [review]
some basic get rid of comments

File: frontend/src/screens/Game.tsx [hunk 4]
```diff
@@ -21,6 +22,7 @@ export const Game = () => {
         }
         socket.onmessage = (event) => {
             const message = JSON.parse(event.data);
+            console.log(message);
```
Comment (hkirat) [inline frontend/src/screens/Game.tsx:25]: get rid of log please

File: backend1/src/GameManager.ts [hunk 5]
```diff
@@ -21,15 +21,25 @@ export class GameManager {
     }
 
     removeUser(socket: WebSocket) {
+        console.log("inside removeuser")
```
Comment (hkirat) [inline backend1/src/GameManager.ts:24]: same here

**Comment by nimit9** [comment]
I'd done some of this logic to implement joining a game, can you modify your change to add on top of that?

**Pull request closed without merging**